src/lib/render-helpers.ts

```diff
@@ -0,0 +1,71 @@
+/**
+ * @license
+ * Copyright (c) 2017 The Polymer Project Authors. All rights reserved.
+ * This code may only be used under the BSD style license found at
+ * http://polymer.github.io/LICENSE.txt
+ * The complete set of authors may be found at
+ * http://polymer.github.io/AUTHORS.txt
+ * The complete set of contributors may be found at
+ * http://polymer.github.io/CONTRIBUTORS.txt
+ * Code distributed by Google as part of the polymer project is also
+ * subject to an additional IP rights grant found at
+ * http://polymer.github.io/PATENTS.txt
+ */
+
+import {camelToDashCase} from '@polymer/polymer/lib/utils/case-map.js';
```

Comment:
Is the plan to copy/paste or reimplement here and remove this dependency on Polymer (along with the declared dependency in `package.json`)?